src/software/backend/input/network/network_client.cpp

```diff
@@ -0,0 +1,228 @@
+#include "software/backend/input/network/network_client.h"
```

Comment:
:exclamation: Copied from software/backend/input/network/networking with some refactoring and new functions